google-cloud-tasks/test/google/cloud/tasks/v2/cloud_tasks_client_test.rb

```diff
@@ -702,12 +702,12 @@ def updater_proc
 
     it 'invokes get_iam_policy with error' do
       # Create request parameters
-      formatted_resource = Google::Cloud::Tasks::V2::CloudTasksClient.queue_path("[PROJECT]", "[LOCATION]", "[QUEUE]")
+      resource = ''
```

Comment:
I'm not sure this change was intended for tests as well as docs.

Reply:
I pulled this branch and ran `rake ci`, and it passes. But I'm still not sure this value is appropriate for tests.

Reply:
@blowmage @dazuma Do you think these empty string values  (typically seen in docs) were also intended for tests?

Reply:
This is okay. It's unfortunate that we lose some flavor in the tests, but when we get rid of the old gapic configs in favor of proto annotations, we're losing some information.

Reply:
Also, this is the low-level API for trace.

Reply:
Thanks!